runner/controller.go

```diff
@@ -0,0 +1,16 @@
+package runner
+
+// Controller controls runs, implementing tenancy and queuing (or not).
+type Controller interface {
```

Comment:
So this is the piece that i'm suggesting should be fundamentally different, whether its running on the worker or via local scoot daemon.  And if you want this interface for the Local Scoot Daemon it should only be part of that code base.  

The worker isn't going to wait a short period of time for the command to finish.  I think this is an extra layer that is not needed, and should be up to the worker code & scoot daemon code independently to decide how to call the runner. 

I suggest common functionality like checking if a command is well formed should be pulled out into a utility library.  (this could go in the runner package).